spec/client_spec.rb

```diff
@@ -20,6 +20,7 @@
                             liqui
                             gemini
                             hitbtc
+                            bittrex
```

Comment:
yeah I think I'll take the @tmlee mentioned in Gitter, just stub it and see that method works or not